src/components/Navigation/Navigation.js

```diff
@@ -24,6 +32,12 @@ import LoginModal from './LoginModal';
 
 
 class Navigation extends React.Component {
+  static propTypes = {
+    isLoggedIn: PropTypes.bool,
+    toggleFunc: PropTypes.func,
```

Comment:
'toggleFunc' doesn't give any clue about what it does. A more suitable name would be 'toggleDrawer'.